### DISCUSSION TITLE
Cloud storage integration (gdrive, droppox, etc)

### DISCUSSION THREAD

**Discussion by cdguy** [rejetto/hfs#839] [Ideas]
I use this repo in my local network and I must say I love it!

it wouls be great if we could integrate network storage services like gdrive, mega, dropbox, etc.

thanks!

**Comment by rejetto** [comment]
hey!
while this is technically possible, it's not currently in the plan.
your best option is to find something to integrate those services in your file system, then you'll be able to share it with hfs.
